exec-service-wasmer/src/wasmer_opcode_control.rs

```diff
@@ -146,6 +146,25 @@ impl FunctionOpcodeControl {
         self.breakpoints_middleware
             .inject_breakpoint_condition(state, BREAKPOINT_VALUE_MEMORY_LIMIT);
     }
+
+    fn inject_memory_grow_check<'b>(&self, state: &mut MiddlewareReaderState<'b>) {
+        self.inject_memory_grow_limit_check(state);
+        self.inject_memory_grow_count_increment(state);
```

Comment:
The "private" globals used by this middleware should be protected in the same way the Metering middleware protects the gas limit and gas counter.

Reply:
good catch

Reply:
will address in #13